pretrained_models/models.py

```diff
@@ -20,10 +21,16 @@ def get_model(model_name):
         return keras.applications.ResNet50V2(weights="imagenet", include_top=False)
     if model_name == "inceptionV3":
         return keras.applications.InceptionV3(weights="imagenet", include_top=False)
-    if model_name == "googlenet":
+    if model_name == "inceptionV1":
         tf.compat.v1.keras.backend.set_image_data_format('channels_first')
         return googlenet.create_googlenet()
     if model_name == "inceptionV1slim":
         inputs = tf.random.uniform((1, 224, 224, 3), dtype=tf.dtypes.float32)
         return slim_nets.inception.inception_v1(inputs)
-    return keras.applications.VGG16(weights="imagenet", include_top=False)
+    if model_name == "VGG16":
+        return keras.applications.vgg16(weights= "imagenet", include_top=False)
+    if model_name == "alexnet":
+        return alexnet.AlexNet()
```

Comment:
This does not use a `weights_path`, is that intended? For other models, we load the pretrained weights by default.

Reply:
I wrote the initialization of the weight_path inside the main of alexnet.py. But if we want to get rid of it, I will give the pretrained weight as default in the model.py file.